README.md

```diff
@@ -14,18 +14,18 @@ The classical Ultrablue control flow consists of several steps:
 To enroll a phone as a verifier, start the server in enroll mode. This will display a QR code on the terminal. From the phone, run the client app, and tap the **+** icon on the top right corner to show a QR code scanner. On scan, an encrypted Bluetooth Low Energy channel will be established, and the enrollment will run automatically. Upon success, a device card will appear on the home page of the client application.
 
 ### 2. Initramfs configuration (optional)
-The following explains how to integrate Ultrablue in the boot process to guard disk encryption by a secret delivered by the phone.
+Once enrolled, you have to re-generate your initramfs in order to include the **ultrablue dracut module** in it, it can be done using the following dracut command:
 
+```
+dracut --add ultrablue /path/to/initrd --force
```

Comment:
That will only work if the ultrablue module is somehow installed somewhere dracut can find it, right? Maybe we should mention how to do that, or how to use a relative path and point to the module in the git repo.

Reply:
The user have to install the ultrablue dracut module, we could  indeed explain how to do it here